### PR TITLE
bump linux kernel to 5.10 and containerd to 1.5.2

### PR DESCRIPTION
**Pull request by chrischdi**

### CODE REVIEW

**Review by seanschneeweiss — APPROVED** [review]

**Review by tobiasgiese — APPROVED** [review]

File: images/capi/ansible/roles/dhc/tasks/debian.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     install_recommends: yes
```
Comment (seanschneeweiss) [inline images/capi/ansible/roles/dhc/tasks/debian.yml:7]: Line 2:
```diff
- - name: Install HWE Kernel 5.8
+ - name: Install OEM Kernel 5.10
```

**Review by seanschneeweiss — DISMISSED** [review]

**Review by johannesfrey — APPROVED** [review]

**Comment by JensErat** [comment]
This includes containerd built with golang 1.16, right? -> silence for high memory usage can be removed after rollout

**Comment by chrischdi** [comment]
Yes, since 1.5.1 they are using Go 1.16.4.

However: we may drop the kernel change as it is not resolving the slab issue as of now.

**Comment by tobiasgiese** [comment]
> Yes, since 1.5.1 they are using Go 1.16.4.
> 
> However: we may drop the kernel change as it is not resolving the slab issue as of now.

I think Jens meant the `SystemdUnitHighMemory` silence on `containerd.service`

**Comment by JensErat** [comment]
> I think Jens meant the `SystemdUnitHighMemory` silence on `containerd.service`

Yes, that's what I was referring to. We can remove that silence with 1.21 rollout. Is there a list of rollout stuff you're tracking such tasks?

**Comment by tobiasgiese** [comment]
> > I think Jens meant the `SystemdUnitHighMemory` silence on `containerd.service`
> 
> Yes, that's what I was referring to. We can remove that silence with 1.21 rollout. Is there a list of rollout stuff you're tracking such tasks?

Yes, issue `#4315` in `t1`. I'll add it.

**Review by bavarianbidi — APPROVED** [review]
but please adjust the name of the playbook step (sean already requested this change)

**Review by tobiasgiese — APPROVED** [review]

**Pull request merged**